pkgs/development/python-modules/pleroma-bot/default.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, oauthlib
+, requests_oauthlib
+, requests
+, pyaml
+}:
+
+buildPythonPackage rec {
+  pname = "pleroma-bot";
+  version = "0.8.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "85056df457a49f2adbd36349d821c9401003d91a753de6456ceb805770e38c13";
+  };
+
+  doCheck = false;
+  propagatedBuildInputs = [ pyaml requests requests_oauthlib oauthlib ];
+
+  meta = with lib; {
+    homepage = "https://github.com/robertoszek/pleroma-bot";
```

The upstream project has a separate documentation website in addition to their GitHub presence.

```suggestion
    homepage = "https://robertoszek.github.io/pleroma-bot/";
```

---

I wasn't completely sure what the ```homepage``` value should've been in this case. This clears things up, thanks.